components/bl-calendarHeatmap-component/src/helpers.js

```diff
@@ -1,9 +1,33 @@
-export const validate = items => {
-  if (typeof items === 'string') {
-    return items.split(',');
+import { today } from './index';
+
+export const generateData = (numberDays, data) => {
+  if (numberDays && data) {
+    const newData = [];
+
+    for (let i = 0; i < numberDays; i++) {
+      const previousDay = getPreviousDay(today, i);
+      const foundData = data.find(value => new Date(value.date).toDateString() === previousDay.toDateString());
```

Comment:
```suggestion
      const foundData = data.find(({date}) => new Date(date).toDateString() === previousDay.toDateString());
```

Also, why we compare strings? How about `valueOf`?
Also it could be utility function - `isSameDate` for example

Reply:
_Also, why we compare strings? How about valueOf?_
valueOf is not suitable because valueOf includes time